ui/Decoder.elm

```diff
@@ -13,7 +13,7 @@ buildRecordDecoder =
         |> required "build" buildDecoder
         |> required "download" downloadDecoder
         |> required "source" sourceDecoder
-        |> required "systemaddons" (nullable (list systemAddonDecoder))
+        |> required "systemaddons" (maybe (list systemAddonDecoder))
```

Comment:
What's the point of making it `maybe` instead of `nullable`? Isn't the result the same in this context?